guides/common/modules/proc_deleting-multiple-content-view-versions.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Deleting_Multiple_Content_View_Versions_{context}"]
+= Deleting Multiple Content View Versions
+
+You can delete multiple content view versions simultaneously.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Content View*.
+. Click the content view you want to delete versions of.
+. Navigate to the *Versions* tab.
+. Select the check box of the version or versions you want to delete. 
+. Click the three dots icon at the top of the list of content views.
```

Comment:
"Three dots icon" has a name, I think it might be the same as we were discussing recently here: https://github.com/theforeman/foreman-documentation/pull/1226#discussion_r860133777

Reply:
Hi Lena, I'll do that. There are also a couple of instances where this is referred to as three dots. I'll fix that and push another PR for it.